set_plugin_dates.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+cd plugins || exit 1
+for d in */ ; do
+  cd "$d" || exit 1
+
+  for VERSION_DIR_NAME in */ ; do
+    # Remove trailing slash
+    VERSION_DIR_NAME=${VERSION_DIR_NAME%/}
+    cd "${VERSION_DIR_NAME}" || exit 1
+
+    DATE=$(date -I)
+    yq w meta.yaml firstPublicationDate "${DATE}" -i || exit 1
```

Comment:
Can you elaborate what this command does? I'm not familiar with the syntax of yq and it seems that we overwrite dates on each build of the image.

Reply:
yes, as of now, these dates will be generated at built of docker image, kind of a placeholder

Reply:
We can check if the field is missing then add it. It would pretty accurate. But would work for the first publication date only.